packages/admin/resources/views/components/layouts/app.blade.php

```diff
@@ -20,7 +20,7 @@ class="filament-sidebar-close-overlay fixed inset-0 z-20 w-full h-full bg-gray-9
                 x-data="{}"
                 x-bind:class="{
                     'lg:pl-[5.4rem] rtl:lg:pr-[5.4rem]': ! $store.sidebar.isOpen,
-                    'lg:pl-[var(--sidebar-width)] rtl:lg:pr-[var(--sidebar-width)]': $store.sidebar.isOpen,
+                    'filament-sidebar-open lg:pl-[var(--sidebar-width)] rtl:lg:pr-[var(--sidebar-width)]': $store.sidebar.isOpen,
```

Comment:
What's the goal of this class? It shouldn't be the same as the one on the sidebar, right?

Reply:
It doesn't have to be the same, but it's doesn't matter since the css can target either with .filament-sidebar.filament-sidebar-open and .filament-main.filament-sidebar-open.

This allows both to be accessed and modified base on the opened/closed state of the sidebar.

Reply:
Yeah but if someone adds styling for `.filament-sidebar-open` it would also style `.filament-main`, which is unexpected.

Reply:
Done.

Reply:
Thanks! LGTM